### ISSUE TITLE
can't build boost 1.75.0 without c++11 because of b2 requires C ++11

### ISSUE THREAD

**Issue by gycherish** [boostorg/boost#485]

**Comment by mclow** [comment]
Yes, you can. You just need C++11 support to build b2. Once that is done, you can build all the rest of boost with C++03. (Ok, most of boost; some of the libraries require C++11 or 14 or 17)
